util/encryption.go

```diff
@@ -0,0 +1,132 @@
+package util
+
+import (
+	"bytes"
+	"crypto/aes"
+	"crypto/cipher"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+var (
+	// "the key" to encrypt/decrypt passwords with
+	key        = os.Getenv("ENCRYPTION_KEY")
+	keyPresent = false
+
+	// regex to pull the actual secret out of the miq-password style column
+	xtractRe = regexp.MustCompile(`v2:{(.*)}`)
```

Comment:
The following string would be valid with the defined regex: `v2:{()}`. Maybe that `.*` needs to be `.+`?

Reply:
Oops, yeah, I forgot about the capturing groups :sweat_smile: